api/db/setup_local_database.sh

```diff
@@ -41,7 +39,7 @@ trap finish EXIT
 cat drop_db.sql create_db.sql | envsubst > $CREATE_DB_FILE
 
 echo "Creating empty database..."
-mysql -u "$ROOT_DB_USER" $ROOT_PASSWORD_ARGS < ${CREATE_DB_FILE}
+mysql -u "$ROOT_DB_USER" -p${ROOT_PASSWORD} < ${CREATE_DB_FILE}
```

Comment:
Though I don't know why this is necessary, exactly. @blrubenstein can you explain?